stable/jenkins/templates/config.yaml

```diff
@@ -117,7 +117,7 @@ data:
     echo "false" > /usr/share/jenkins/ref/secrets/slave-to-master-security-kill-switch;
     cp -n /var/jenkins_config/config.xml /var/jenkins_home;
 {{- if .Values.Master.InstallPlugins }}
-    cp -n /var/jenkins_config/plugins.txt /var/jenkins_home;
+    cp /var/jenkins_config/plugins.txt /var/jenkins_home;
```

Comment:
I wanted to keep everything in one place so it was easy to see what was going on.